src/app/views/contributions.py

```diff
@@ -0,0 +1,23 @@
+from django.views.generic.base import TemplateView
+from app.models import Contribution
+
+
+class Contributions(TemplateView):
+    template_name = "contributions.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        contributions = Contribution.objects.all().order_by('-created', '-id')
```

Comment:
massive TODO: stop using `.all` and implement paging! Or filter it somehow.

Reply:
ya, all of this needs to be changed, there's already a ticket for this :)